Fix screen rotation issue in permission activity layout

AgarwalsRahul opened the pull request:
Fix the rotation issue by adding a scroll view in the permission activity

AgarwalsRahul committed:
Fix screen rotation issue in permission activity layout

prathameshmm02 commented:
Instead of adding the buttons inside a ScrollView, removing the ScrollView & changing the orientation of LinearLayout to Horizontal in Landscape mode would be a better alternative in my opinion. 
So basically,
Portrait -> Vertical orientation
Landscape -> Horizontal orientation
But that would require creating new layout file for Landscape configuration which is redundant as we are just dealing with one property, orientation.
So you can follow a similar approach I did [here](https://github.com/RetroMusicPlayer/RetroMusicPlayer/blob/17237a4021a2fec31300b778606248c73998a7f0/app/src/main/res/layout/fragment_main_recycler.xml#L22) i.e. store a integer resource for the value of that property. 
Let me know if you have any doubts about this.

AgarwalsRahul commented:
@prathameshmm02 I removed the scroll view and changed the orientation to horizontal using integer resource but still, both of these permissions are not fully visible. Should I add a scroll view?
![Screenshot](https://user-images.githubusercontent.com/63352614/156477301-d426cb74-c851-4fe0-9d85-e778a76126ac.jpg)

prathameshmm02 commented:
Yeah okay, go ahead.

AgarwalsRahul commented:
@prathameshmm02 Can u check if it's fine?

[!Video](https://user-images.githubusercontent.com/63352614/156488854-d6ba32d2-a976-440a-82d2-a29151df0d0c.mp4)

prathameshmm02 commented:
That's great, but you haven't pushed your changes.

AgarwalsRahul committed:
Fix Orientation Issue
